make fail (`cmd/nerdctl/run_network.go:33:2: //go:build comment without // +build comment`)

after this commit https://github.com/containerd/nerdctl/commit/d4fb38c4701e68110cb52185fbb95ab61b7f1d59 when use golang 1.16
```
cmd/nerdctl/run_network.go:33:2: //go:build comment without // +build comment
make: *** [Makefile:50：nerdctl] 
```

--- a/README.md
+++ b/README.md
@@ -201,6 +201,8 @@ information in our [`containerd/project`](https://github.com/containerd/project)
 
 ### Compiling nerdctl from source
 
+**Tips: You should make your Go toolchain newer than 1.17.**
+
 Run `make && sudo make install`.
 
 Using `go install github.com/containerd/nerdctl/cmd/nerdctl` is possible, but unrecommended because it does not fill version strings printed in `nerdctl version`

